scenes/game/local_game.gd

```diff
@@ -1724,7 +1729,7 @@ class Player:
 				if not parent.game_over:
 					var card = deck[0]
```

Comment:
you didn't change which card was pulled off though?
Maybe in a later commit you fix this, i'm just going through them in order.

Reply:
okay, looks like you got it in the latest commit